app/policies/application_policy.rb

```diff
@@ -29,7 +29,13 @@ class Scope
     attr_reader :user, :scope
     def resolve
       return scope.all if @user.role?("admin") || @user.role?("manager")
-      return scope.where(draft: false) if scope.column_names.include?("draft")
+      if @user.role?("analyst")
```

Comment:
looks like this does the right thing (https://github.com/dumparkltd/marine-defrag-server/pull/56#issuecomment-957252451):

if tables can have draft content, it is only visible to analysts when it is not draft.
if tables can NOT have draft content, analyst can see it

Reply:
Awesome 😄